pkg/networkservice/acl/common.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package acl
+
+import (
+	"context"
+	"time"
+
+	"git.fd.io/govpp.git/api"
+	"github.com/edwarnicke/govpp/binapi/acl"
+	"github.com/edwarnicke/govpp/binapi/acl_types"
+	"github.com/edwarnicke/govpp/binapi/interface_types"
+	"github.com/pkg/errors"
+
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+)
+
+const (
+	aclTag = "nsm-acl-from-config"
+)
+
+func create(ctx context.Context, vppConn api.Connection, tag string, swIfIndex interface_types.InterfaceIndex, aRules []acl_types.ACLRule) ([]uint32, error) {
+	interfaceACLList := &acl.ACLInterfaceSetACLList{
+		SwIfIndex: swIfIndex,
+	}
+
+	var err error
+	interfaceACLList.Acls, err = addACLToACLList(ctx, vppConn, tag, false, aRules)
+	if err != nil {
+		log.FromContext(ctx).Info("ACL_SERVER: error adding acl to acl list ingress")
```

Comment:
Please use here `.WithField` instead of manually adding `ACL_SERVER:` in every log message:
```go
func create(...) ... {
	logger := log.FromContext(ctx).WithField("acl", "create")
	...
	logger.Info(...)
	...
}
```

Reply:
done